examples/home/flake.nix

```diff
@@ -0,0 +1,46 @@
+{
+  inputs = {
+    # Principle inputs (updated by `nix run .#update`)
+    nixpkgs.url = "github:nixos/nixpkgs/nixos-unstable";
+    home-manager.url = "github:nix-community/home-manager";
+    home-manager.inputs.nixpkgs.follows = "nixpkgs";
+
+    flake-parts.url = "github:hercules-ci/flake-parts";
+    nixos-flake.url = "github:srid/nixos-flake";
+  };
+
+  outputs = inputs@{ self, ... }:
+    inputs.flake-parts.lib.mkFlake { inherit inputs; } {
+      systems = [ "x86_64-linux" "aarch64-darwin" "x86_64-darwin" ];
+      imports = [
+        inputs.nixos-flake.flakeModule
+      ];
+
+      flake =
+        let
+          # TODO: Change username
+          myUserName = "john";
+        in
+        {
+          homeConfigurations.${myUserName} =
+            self.nixos-flake.lib.mkHomeConfiguration
+              "x86_64-linux"  # FIXME: how to not hardcode this?
```

Comment:
The trick I found is to use ` legacyPackages.homeConfigurations` in `perSystem`, see https://github.com/edrex/nixcfg/blob/main/flake.nix#L41 and https://github.com/nix-community/home-manager/issues/3075#issuecomment-1477155995

Reply:
Thanks! @roberth also pointed in this direction.